cypress/e2e/brewery/DatasetManager.cy.js

```diff
@@ -124,6 +124,8 @@ describe('Data edition in dataset manager', () => {
 });
 
 describe('Dataset creation', () => {
+  const datasetAuthor = 'Dave Lauper';
```

Comment:
Instead of hard-coding the user name in tests, we can use the constant `USER_EXAMPLE` defined in the stubbing data file: _cypress/fixtures/stubbing/default/users.js_